### PR TITLE
Document real partial mock

### PR DESCRIPTION
**Pull request by derlin**
Add a section about calling the original method. #436.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,6 +272,25 @@ every { func() } returns Car() // or you can return mockk() for example
 func()
 ```
 
+### Real partial (relaxed) mock
```
Comment (Raibaz) [inline README.md:275]: I would just call this section "Partial mocking"

**Review by Raibaz — CHANGES_REQUESTED** [review]

**Comment by Raibaz** [comment]
Thanks for sending this through! I just have a minor concern on the title of the section.

File: README.md (hunk 1 above)
Comment (derlin) [reply README.md:275]: Had the same feeling, thinking about renaming it to "*calling real method*" instead, but I also like your suggestion. Sold !

**Comment by Raibaz** [comment]
Awesome, thanks a lot!

**Pull request merged**